module/questions/helper.js

```diff
@@ -632,7 +678,15 @@ module.exports = class QuestionsHelper {
    */
 
   static booleanData() {
-    let booleanData = ["allowAudioRecording", "showRemarks", "isAGeneralQuestion", "autoCapture"];
+    let booleanData = [
+      "allowAudioRecording", 
+      "showRemarks", 
+      "isAGeneralQuestion", 
+      "autoCapture",
+      "prefillFromEntityProfile",
+      "canEdit"
```

Comment:
@aman-tunerlabs - Can we rename it to isEditable ? Also rather than hardcoding can we get all fields from model of type boolean, so need to update this function every time